### PR TITLE
Simple Parser to experiment with and learn from (second attempt)

### PR DESCRIPTION
**Pull request by ordovician**
This is a very simple parser, for people who want to learn and experiment with parsers. I got a module reading and writing NeXTSTEP style plist files using this parser I hope to add as a module as well later.

### CODE REVIEW

File: SimpleParser/versions/0.0.1/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+6185fecf542ef7e92b7b68c4777ee081773f96de
```
Comment (tkelman) [inline SimpleParser/versions/0.0.1/sha1]: probably best to not register this tag since it doesn't have a minimum julia version

**Comment by aviks** [comment]
Hi @ordovician if this is still relevant, could you please tag this correctly with a minimum julia version?

**Comment by KristofferC** [comment]
Closing due to inactivity.

**Comment by ordovician** [comment]
Hi Avik, I am on vacation now but I’ll have a look when I am home.

Erik

**Pull request closed without merging**